apps/emqx_auth_http/etc/emqx_auth_http.conf

```diff
@@ -151,3 +151,17 @@ auth.http.pool_size = 32
 ##
 ## Value: File
 ## auth.http.ssl.keyfile = {{ platform_etc_dir }}/certs/client-key.pem
+
+## In mode verify_none the default behavior is to allow all x509-path
+## validation errors.
+##
+## Value: true | false
+## auth.http.ssl.verify = false
+
+## Specify the hostname to be used in TLS Server Name Indication extension.
+## If not specified it will default to the Host argument of connect/[3,4] unless it is of type inet:ipaddress().
+## The HostName will also be used in the hostname verification of the peer certificate.
+## The special value disable prevents the Server Name Indication extension from being sent and disables the hostname verification
+##
+## Value: String | disable
+## auth.http.ssl.server_name_indication = disable
```

Comment:
maybe change to `auth.http.ssl.verify_hostname`

Reply:
In fact, it is a standard name. Modifying it may make it difficult to understand.